src/coredump/coredumpctl.c

```diff
@@ -1120,9 +1120,13 @@ static int run_debug(int argc, char **argv, void *userdata) {
         if (r < 0)
                 goto finish;
         if (r == 0) {
+                int saved_errno;
+
                 execvp(debugger_call[0], debugger_call);
+
+                saved_errno = errno;
```

Comment:
hmm, I'd just do this in log_open(). We do that in log_error_errno(), and I think doing it in lo_open() too would make a lot of sense, given it is called in similar contexts where errno should not be affected. i.e. we should have `PROTECT_ERRNO;` at the top of the log_open() implementaiton I think

Reply:
Yeah, that was the other alternative I've considered (as mentioned above); but was not sure which option you'd prefer.

I've ported everything back to `PROTECT_ERRNO`, PTAL.